implementation/.github/workflows/test-pull-request.yml

```diff
@@ -42,15 +42,26 @@ jobs:
         GIT_TOKEN: ${{ github.token }}
 
   approve:
-    name: Auto Approve
+    name: Approve Bot PRs
     if: ${{ github.event.pull_request.user.login == 'paketo-bot' || github.event.pull_request.user.login == 'dependabot[bot]' }}
     runs-on: ubuntu-latest
     needs: integration
     steps:
+    - name: Check for Human Commits
+      id: human-commits
+      uses: paketo-buildpacks/github-config/actions/pull-request/check@main
+      with:
+        token: ${{ secrets.PAKETO_BOT_REVIEWER_GITHUB_TOKEN }}
+        repo: ${{ github.repository }}
+        number: ${{ github.event.number }}
+        bots: "dependabot[bot],paketo-bot,GitHub"
```

Comment:
Why are these different than the defaults?

Reply:
Related to [this comment](https://github.com/paketo-buildpacks/github-config/pull/219/files#r570508444) I believe.